src/components/autosuggest/_macro.spec.js

```diff
@@ -5,130 +5,177 @@ import * as cheerio from 'cheerio';
-        expect($element.attr('data-aria-you-have-selected')).toBe('You have selected');
-        expect($element.attr('data-autosuggest-data')).toBe('/examples/data/country-of-birth.json');
-        expect($element.attr('data-instructions')).toBe('Use up and down keys to navigate.');
-        expect($element.attr('data-more-results')).toBe('Continue entering to improve suggestions');
-        expect($element.attr('data-no-results')).toBe('No suggestions found. You can enter your own answer');
-        expect($element.attr('data-results-title')).toBe('Suggestions');
-        expect($element.attr('data-type-more')).toBe('Continue entering to get suggestions');
-    });
+            test('THEN: it has expected id on container element', () => {
+                expect($('.ons-autosuggest').attr('id')).toBe('country-of-birth-container');
+            });
 
-    it('has the `data-allow-multiple` attribute when `allowMultiple` is `true`', () => {
-        const $ = cheerio.load(
-            renderComponent('autosuggest', {
-                ...EXAMPLE_AUTOSUGGEST,
-                allowMultiple: true,
-            }),
-        );
+            test('THEN: it has a special class that indicates the component should initialise itself', () => {
+                const $ = cheerio.load(renderComponent('autosuggest', EXAMPLE_AUTOSUGGEST));
+
+                expect($('.ons-autosuggest').hasClass('ons-js-autosuggest')).toBe(true);
+            });
+
+            test('THEN: it has the provided data attributes', () => {
+                const $element = $('.ons-autosuggest');
+                expect($element.attr('data-allow-multiple')).toBeUndefined();
+                expect($element.attr('data-min-chars')).toBe('2');
+                expect($element.attr('data-aria-limited-results')).toBe('Type more characters to improve your search');
+                expect($element.attr('data-aria-min-chars')).toBe('Enter 3 or more characters for suggestions.');
+                expect($element.attr('data-aria-n-results')).toBe('There are {n} suggestions available.');
+                expect($element.attr('data-aria-one-result')).toBe('There is one suggestion available.');
+                expect($element.attr('data-aria-you-have-selected')).toBe('You have selected');
+                expect($element.attr('data-autosuggest-data')).toBe('/examples/data/country-of-birth.json');
+                expect($element.attr('data-instructions')).toBe('Use up and down keys to navigate.');
+                expect($element.attr('data-more-results')).toBe('Continue entering to improve suggestions');
+                expect($element.attr('data-no-results')).toBe('No suggestions found. You can enter your own answer');
+                expect($element.attr('data-results-title')).toBe('Suggestions');
+                expect($element.attr('data-type-more')).toBe('Continue entering to get suggestions');
+            });
+
```

Comment:
These should probably be split out into multiple thens because they are testing individual params a bit like the id test above this